peterdesmet/petridish
# Add heading anchors for h2 through h6.

Fixes peterdesmet/petridish#7

This displays the anchors to the left of the headings, on hover, and uses your default link styles.

It won't show on `.card-title` headings or the headings on the team page, but it should show for all other headings from h2 to h6.

**To test it out:**
1. Pull down the branch and run `npm run copy:all`
2. Start the server, browse to `/about`, and hover over the headings to see how it looks.

Let me know if you want to adjust anything. 😄
